src/ODataHttpClient/ODataHttpClient.csproj

```diff
@@ -8,8 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="[5.2.4,)" />
-    <PackageReference Include="Newtonsoft.Json" Version="[10.0.1,)" />
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="[5.2.7,)" />
+    <PackageReference Include="Newtonsoft.Json" Version="[12.0.3,)" />
```

Comment:
@xin9le 

Current:
Newtonsoft.Json >= 10.0.1

Your PR
Newtonsoft.Json>= 12.0.3

Would you tell me why you have raised the version range?